jaxb-ri/bundles/runtime/pom.xml

```diff
@@ -250,6 +250,8 @@
                             <artifactSet>
                                 <includes>
                                     <include>org.glassfish.jaxb:jaxb-runtime</include>
+                                    <include>org.glassfish.jaxb:txw2</include>
+                                    <exclude>com.sun.istack:istack-commons-runtime</exclude>
```

Comment:
this should say include instead

Reply:
Indeed, interesting that plugin did eat it.